docs.md

```diff
@@ -56,7 +56,7 @@ clone:
 | `home`                    |                                     | Change HOME var for commands executed, fail if it does not exist                                                                                                           |
 | `remote`                  | $CI_REPO_CLONE_URL                  | Set the git remote url                                                                                                                                                     |
 | `remote-ssh`              | $CI_REPO_CLONE_SSH_URL              | Set the git SSH remote url                                                                                                                                                 |
-| `object-format`           | sha1              | Set the object format for Git initialization. Supported values: sha1, sha256. |
+| `object-format`           | detected from commit SHA            | Set the object format for Git initialization. Supported values: `sha1`, `sha256`.                                                                                              |
```

Comment:
actually, it did not detected in my case, but may be it was some old plugin version.